README.md

```diff
@@ -4,33 +4,34 @@ GitHub action to automatically publish packages to npm.
 
 ## Usage
 
-Add this to your `.github/main.workflow` file:
-
-```
-workflow "publish to npm on push" {
-  on = "push"
-  resolves = ["npm publish"]
-}
-
-action "npm publish" {
-  uses = "pascalgn/npm-publish-action@6fff82ba2c6462bf4f6940168afe65303bbdbd56"
-  secrets = ["GITHUB_TOKEN", "NPM_AUTH_TOKEN"]
-  env = {
-    TAG_NAME = "v%s"
-    TAG_MESSAGE = "v%s"
-    COMMIT_PATTERN = "^Release (\\S+)"
-  }
-}
+Add a step like this to your workflow:
+
+```yaml
+- name: Publish if version has been updated
+  uses: pascalgn/npm-publish-action@1.0.1
```

Comment:
This should always use the full commit sha, for security reasons!

Reply:
The standard way GitHub tells to use actions is by using published releases, I think it would be super-annoying for people to have to copy the whole sha over and over. Even on the Marketplace if you click the "Use latest version" button it gives you the release reference

Reply:
People are free to use published versions or even the branch name, but I don't want to recommend it in the docs. Published versions can be removed and re-created and until GH provides a way to mark releases as "never allow deletion/updates" (which they probably will in the future), using the git hash is much safer

Reply:
Ok. I can't do it though, because it needs to be done after this PR is merged, so that we can get the latest commit's sha. You can do it like this: merge the PR, add the sha of that commit to the readme, then publish this last commit as the new version: it won't technically be the same commit, but the only difference will be in the readme, which doesn't affect how the action works.